src/client/components/localisation/skybox/view_model.ts

```diff
@@ -0,0 +1,90 @@
+import { createTransformer } from 'mobx'
+import { computed } from 'mobx'
+import { BackSide } from 'three'
+import { Mesh } from 'three'
+import { SphereBufferGeometry } from 'three'
+import { MeshBasicMaterial } from 'three'
+import { PlaneBufferGeometry } from 'three'
+import { ShaderMaterial } from 'three'
+import { UniformsUtils } from 'three'
+import { Vector3 } from 'three'
+import { SkyboxModel } from './model'
+import { SkyShader } from './sky_shader'
+
+export class SkyboxViewModel {
+
+  public constructor(private model: SkyboxModel) {
+  }
+
+  public static of = createTransformer((model: SkyboxModel): SkyboxViewModel => {
+    return new SkyboxViewModel(model)
+  })
+
+  @computed
+  public get skybox() {
+    // reference: http://threejs.org/examples/#webgl_shaders_sky
+    const shader = new SkyShader()
+    const uniforms = UniformsUtils.clone(shader.uniforms)
```

Comment:
Shouldn't the uniforms values come from the model? I think we can delete the `SkyShader` class and just inline the values directly here.

e.g.

```
const mat = new ShaderMaterial({
  fragmentShader: SkyboxFrag,
  vertexShader: SkyboxVert,
  uniforms: {
    luminance: { value: this.model.luminance },
    etc...
  },
  side: BackSide,
})
```